atef/widgets/config/window.py

```diff
@@ -296,6 +301,40 @@ def print_report(self, *args, **kwargs):
 
         run_tree.print_report()
 
+    def clear_results(self, *args, **kwargs):
+        """ clear results for a given tree """
+        current_tree = self.tab_widget.currentWidget().get_tree()
+
+        if isinstance(current_tree, RunTree):
+            config_file = current_tree.config_file
+            # ask for confirmation first
+            reply = QMessageBox.question(
+                self,
+                'Confirm deletion',
+                (
+                    'Are you sure you want to clear the results of the checkout: '
+                    f'{config_file.root.name}?'
+                ),
+            )
+            if reply != QMessageBox.Yes:
+                return
+
+            if isinstance(config_file, ProcedureFile):
```

Comment:
Nitpick: seems like this should be in a `clear_results` helper function outside of the GUI code?

Reply:
done in e2d4988